.github/workflows/databend-release.yml

```diff
@@ -94,7 +94,7 @@ jobs:
       - name: Set musl rustflags
         if: matrix.platform == 'musl'
         run: |
-          flags="-C link-arg=-Wl,--compress-debug-sections=zlib-gabi"
```

Comment:
`--compress-debug-sections=zlib-gabi` is still needed?

Reply:
I think it won't affect the build. Remove it temporarily, x86_64 does not seem to recognise it correctly.